lib/appdmg.js

```diff
@@ -387,7 +387,8 @@ module.exports = exports = function (options) {
     ds.setWindowSize(global.bkgsize[0], global.bkgsize[1])
 
     global.opts.contents.forEach(function (e) {
-      ds.setIconPos(path.basename(e.path), e.x, e.y)
+      var p = typeof e.title === 'undefined' ? path.basename(e.path) : e.title
+      ds.setIconPos(path.basename(p), e.x, e.y)
```

Comment:
path.basename is called twice when not using title. Does it make sense to call `basename` on `title`? I don't think we should allow `/` in the `title`.